integration/internal/container/ops.go

```diff
@@ -205,3 +206,10 @@ func WithExtraHost(extraHost string) func(*TestContainerConfig) {
 		c.HostConfig.ExtraHosts = append(c.HostConfig.ExtraHosts, extraHost)
 	}
 }
+
+// WithPlatform specifies the desired platform the image should have.
+func WithPlatform(p *specs.Platform) func(*TestContainerConfig) {
```

Comment:
where is this used?

Reply:
It's not, I added it while I was iterating on the test but ended up not needing it.